### PR TITLE
Add support for min_os_version and max_os_version

### PR DESCRIPTION
**Pull request by tmancey**
Add support for min_os_version and max_os_version to Griffin

### CODE REVIEW

**Review by iefremov — APPROVED** [review]

**Review by moritzhaller — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
